internal/olm/operator/bundle/install.go

```diff
@@ -71,6 +71,11 @@ func (i *Install) setup(ctx context.Context) error {
 		}
 	}
 
+	//if user set --skip-tls then set --skip-tls-verify to true as --skip-tls is deprecated
```

Comment:
I think comment is now out of date. 
```suggestion
	/ /if user sets --skip-tls then set --use-http to true as --skip-tls is deprecated
```

Reply:
Updated